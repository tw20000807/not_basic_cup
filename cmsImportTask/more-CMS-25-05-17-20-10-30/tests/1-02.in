xBKhCirDv7dWDvJlxKTtuZimwF4Zi9x1
1
1 2
482189600
823034429 1655304198 482189600
