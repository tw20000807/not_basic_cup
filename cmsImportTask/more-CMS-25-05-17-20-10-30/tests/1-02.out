du3W59OSqKqC9FuBgxi0VkGO13qNOHq6
823034429 1655304198
